### PR TITLE
add Chinese support

### PR DESCRIPTION
**Pull request by pexcn**
It is OK!

### CODE REVIEW

File: SGit/res/values-zh-rCN/arrays.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <item>改变基点</item>
+        <item>择取</item>
+        <item>比较</item>
+        <item>新建文件</item>
+        <item>新建文件夹</item>
+        <item>远程</item>
+        <item>删除</item>
+    </string-array>
+
+</resources>
```
Comment (d10) [inline SGit/res/values-zh-rCN/arrays.xml]: Missing newline?

File: SGit/res/values-zh-rCN/strings.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+    <string name="progress_left_init">0%</string>
+    <string name="progress_right_init">0/0</string>
+    <string name="push_msg_init">正在推送...</string>
+    <string name="title_activity_private_key_manage">管理私钥</string>
+    <string name="hello_world">Hello, World!</string>
+    <string name="title_activity_explore_file">浏览文件</string>
+
+    <!-- toast msg -->
+    <string name="feedback_subject">反馈</string>
+</resources>
```
Comment (d10) [inline SGit/res/values-zh-rCN/strings.xml]: Same.

File: SGit/res/values-zh-rCN/strings_error.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <string name="alert_no_items_selected">没有选中条目</string>
+    <string name="alert_no_older_commits">没有比较旧的提交来对比</string>
+    <string name="alert_please_wait_previous_op">之前的操作仍然在运行中</string>
+    <string name="alert_you_must_choose_one_commit_to_copy">请选择一个提交来复制</string>
+    <string name="alert_you_should_input_something">请先输入</string>
+    <string name="alert_please_add_a_remote">请先添加一个远程仓库</string>
+    <string name="alert_is_already_a_git_repo">这个目录下已经有一个Git仓库</string>
+    <string name="alert_save_failed">保存失败</string>
+
+</resources>
```
Comment (d10) [inline SGit/res/values-zh-rCN/strings_error.xml]: Again.

**Comment by d10** [comment]
LGTM, except for missing EOF newline.

File: SGit/res/values-zh-rCN/strings_error.xml (hunk 3 above)
Comment (pexcn) [inline SGit/res/values-zh-rCN/strings_error.xml]: But, "values" also like this.

Comment (d10) [inline SGit/res/values-zh-rCN/strings_error.xml]: Ah, must have missed that. Will fix that soon.

Comment (pexcn) [inline SGit/res/values-zh-rCN/strings_error.xml]: OK.

Comment (pexcn) [inline SGit/res/values-zh-rCN/strings_error.xml]: 抱歉，我是新手。
不过你说的对啊，要是不加换行，就跟上完厕所没有洗手似的。。。⊙▽⊙

**Comment by d10** [comment]
@pexcn thanks for the PR! I’ll ping you the next time we need translations, hope you can help out!

**Pull request merged**